### DISCUSSION TITLE
mac office字体无法使用

### DISCUSSION THREAD

**Discussion by longyun200111** [nju-lug/NJUThesis#254] [Q&A]
mac系统上使用office字体会报错，但是我确认了在对应路径下确实有office的字体文件，如果把字体选项改成mac原生字体就能编译成功

<img width="651" alt="image" src="https://github.com/nju-lug/NJUThesis/assets/20951279/8b0b87c3-79b6-4fad-b563-4990f6a7e4f1">

**Comment by atxy-blip** [comment]
这是 <https://github.com/nju-lug/NJUThesis/commit/51f3edde8a1b26d86ba76fee4991ed46fc4bdea3> 修复的问题，可以先用 CI 生成的模板文件 <https://github.com/nju-lug/NJUThesis/actions/runs/8998769758/artifacts/1483338192>。
